<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariantPredicateDialog</class>
 <widget class="QDialog" name="VariantPredicateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add predicate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabPredicateH">
      <attribute name="title">
       <string>Halfspace predicate</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="PredicateH" name="widgetPredicateH" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPredicateS">
      <attribute name="title">
       <string>Screw predicate</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="PredicateS" name="widgetPredicateS" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPredicateG">
      <attribute name="title">
       <string>General predicate</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="PredicateG" name="widgetPredicateG" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PredicateH</class>
   <extends>QWidget</extends>
   <header>src/predicateh.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PredicateS</class>
   <extends>QWidget</extends>
   <header>src/predicates.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PredicateG</class>
   <extends>QWidget</extends>
   <header>src/predicateg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VariantPredicateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VariantPredicateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
